homeassistant/components/sensor/command_line.py

```diff
@@ -90,20 +93,52 @@ def update(self):
 class CommandSensorData(object):
     """The class for handling the data retrieval."""
 
-    def __init__(self, command):
+    def __init__(self, hass, command):
         """Initialize the data object."""
-        self.command = command
         self.value = None
+        self.hass = hass
```

Comment:
Since this is not the entity class, `hass` needs to be passed in. I think @balloob was thinking about the entity class where it could be removed.